central/pruning/pruning.go

```diff
@@ -876,6 +875,6 @@ func (g *garbageCollectorImpl) pruneLogImbues() {
 }
 
 func (g *garbageCollectorImpl) Stop() {
-	g.stopSig.Signal()
-	<-g.stoppedSig.Done()
+	g.stopper.Client().Stop()
+	_ = g.stopper.Client().Stopped().Wait()
```

Comment:
You don't need the left hand `_` assignment. Any reasons to explicitly do that?

Reply:
Same comment for every other left hand empty assignments.

Reply:
`Stopped()` is `ReadOnlyErrorSignal`, therefore its `Wait()` returns an `error`. The `error` will be `nil` if this signal was signalled "normally" or will be an error value if the signal was provided that. For example, in case of abnormal shutdown.

By assigning the result to `_` I calm down GoLand which otherwise tells me that there's a potential `error` that wasn't respected.

With underscore:
![image](https://user-images.githubusercontent.com/537715/208734950-015ad559-e42d-4c8a-a9e5-4433878cfef6.png)

Without underscore:
![image](https://user-images.githubusercontent.com/537715/208735027-36661d05-dbd4-490b-a470-6d46e0a898be.png)